targets/TARGET_NUVOTON/TARGET_M480/trng_api.c

```diff
@@ -89,6 +89,11 @@ int trng_get_bytes(trng_t *obj, uint8_t *output, size_t length, size_t *output_l
             trng_get(output);
             *output_length += 32;
             output += 32;
+        }     
+        if( length > *output_length ) {
+            trng_get(tmpBuff);
+            memcpy(output, &tmpBuff, (length - *output_length));
+            *output_length = length;
```

Comment:
The two code-paths handling the case of remaining length `< 32` can be unified here. Also, while it's a pre-existing defect, we should safely zeroize `tmpBuff` before returning.

Reply:
@hanno-arm , thanks of your comment, added new commit for code unify & zeroize.